csharp/ql/test/query-tests/Security Features/CWE-117/LogForgingSimpleTypes.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Web;
+using Microsoft.Extensions.Logging;
+
+class ILogger
+{
+    public void Warn(string message) { }
+}
+
+enum TestEnum
+{
+    TestEnumValue
+}
+
+public class LogForgingSimpleTypes
```

Comment:
The test code doesn't compile. Consider adding the test code in https://github.com/github/codeql/blob/main/csharp/ql/test/query-tests/Security%20Features/CWE-117/LogForging.cs

Compilation issues:
- Duplicate declaration of `ILogger` (as this class also exists in the file mentioned above).
- Arguments to the `Warn` calls.